.travis.yml

```diff
@@ -3,11 +3,28 @@ services:
+  - secure: f/5CuVYNzwHCGA8ZnYm2tHVMmxU1Cm3u0CaD1ukJwPi1T/btpZ+Pc2muYDIinZVDpdakeRe737TIjGuvQ90WE2zkGyB6jHTWe7G+9ckcBHcCEl9BK40ytHrJlNmAI6qi5ZNBIFPnnTYgnlg/PC6oublP4+K36xGsfSdXG4BMGn+fBse22xSeIKfXQExrZg/rNwJkwV6PuzQvCADrBfJ0tXgHXpu+N2qwCU7PP5VQvOWxlrqptBBXHqCjYriJz2OtacgmJidqTF0nTaBQteMZG/knaYzlN+b9VL+z1kwu9UzdOEuNHMWkuHHQtUzyoaovGNTkT+Cqv/vnGg2rjYqBoM14tpTPhd9WMM8Gy5hfgTt2evcc1CcPpqnm28gUvdKu+1GcibFhYGIeBzOCBCjt2FaW/5fEIYF8eFkZBYJeHiGKGISel7O5T1x5NmTWoixx7boXWdVG2r0AdfHcm2djmChn9NLyAeR5+2AJgaUizkW28j8PGzOtUIBfs+dl0OIwuY1qGBXkSoDIRi+aoe02NCPOnE59K7fR1ETpXvCw1Ff/AslT/hwRLqCF5ik4uvPdTWkpqvaac+Gay9KUorD/rmjUSJuyxxnJYY3pIxzyDLuz9rLMwHJwHT2fa45i5fLINzwutHRvobDqXpb5xRgceIW2Mxut0q9xGXpKO/9eCgc=
+  - secure: YnareXxtYKiEXBTgdZOFaHXqBuTrWkwJ5LhNkX9gZl4Q50wLWPO27dfxpaZZzez1KZWpI1WY3vJRXAwwWax5A0xtNWPNbHQZFCSC4pzNDq1+Z/7Zvj99SQ0vga5gMWPMsedWgg/YihaPV9VEmutMT4pCw5KP9tncpArl67EWxZcpXAICBYaPs9sykkm4dVUNVlhhCyIm2klMQAujPeeXvg0czyo9qnFumpgGfNcgTZrfPXPfASmE0LAF87IVgWLx7phYeLjfdjDuybNyppZ9fw6DEZnEuRmQd6G3RM9ofTibvk0AzXs0E4ubYM5TInstiFb4uNINt2Tf3GDa9W07T8E4BTV0GnxNlme10E5h/Xo0TtgyuFKhyUjFRWCoMou/FCFhW9oZQP7WkIDPYmSENrcBYGVHusKI4NVA3EGjdWUv528q3w58Jg9936+Mc9pCnUtCz7bRiSnSqj0eiZ9fGRpvjsiVNbhezz8BjnN58JbTPkv4xepg0RJMKjg9eIWW/iDsF/9sNr+4ftdLYrQ5LOeEtvnRWXq1x0Zbb0pF5r1KJdvu5z+KEfDIlZz756cO1bsW/MnJIg9OlZuR/Hkzu7I315om8g6UbLFLH3L5X6yiNsL83wzheWwIpZR/8IjVCo3oyi/0KeQXPEldCcGTcys55u4d/rmSDpCdqoLaPCw=
+  - secure: l5VZQZk20vHYknN5zxtE3mLub4SUCPMJ2rZTpqM3BHrgGWdEOQd1o45AHv1wPQURqQNvpTYvOTWe6R6XHEGXwTkOkYrgAL0s3G3XJhB5yf43Qnn1wq9I17a8Mp/Qz26IAQ+pes1wszWzUnUrT/SxSnlNkLjDAADCt/zyflQxi88j3fvd3Wa+3onr4mNaIskSG1KRiPpJa43EA/SeRMG1nZxYV33qD8ekkEi+eb/w74xUI9YsP/AF4I/JexSTxJSPOIFdX06TXy0PetYnpWn2F1rZrvrY0edfcsfdD/CzEimkVqMmDP2p2gWZY96CPTlT/Suqm0ZIEBgQcCZhmmpwG3NiYo4oyLKIkVOvyQT+gABDC4LKy7eJMkqkAHX2gq2OtWpuPxCxqalOlCfHdnx2QBnoOB9w092IJX0ujlbGjjV71e0WalqI+aN81hrVZgJ1fS+u6zuNx90o7HY8iLkDFd8wQkPsku4hCH85NazBgBl1KI5cKBkIoyL9cpUxxmEeEtqgBrQ5fe8FbK5N0p7uwwrCXFPCZAoB0ej3uGkSKr+kgzP0TxKpRo5eTHgyMT1UwUeFRr2D1Pcuc/sQckaGJ5xPa0O3+E8SdrcjyQCRkxAlpCu+l0pvj9HHwHuWmie9rRQOplRR1Cfxg4vbfr6iQpPbGE/J7GKYBj33ryHG3O4=
+  - secure: N+P4cZ/n7i37LetzJ1byXmW4RaiwhRvZ2JAMUY/sSVHCBXYMlyni2fXBdAWxBf33H/+stgt63cfKBUSuBthJBT1Zibgtpk7+11lrKsHDMURv3RB7Z/G7Q1UvGQ4KH9GkFEHL7R+DTguOIddP53HNpSVb4VwQ/6Hz1ePqGPho8vHS7tLBAjxFSKymSeyFnnh6H4A1pHVRXQJ4acAQIMTdE0x9kTLXltWVMdlVWg4QO+ZI1U86hzB7L7CcaRzS6II/56ZtU00LRaiRTxZvIr3xmdCM9RTW9bwKKEigXCZVBnFaUm9UDYlHNwKNgtIB8Rp6QH/QljiOc6FNehnxHaPGBhHzXLL0vaSGQ9GS/qRpKCZ4R/g6VHkZiyXfFVS0+pqn1saVlQ9X1qtj6+LM9GR2bimuvc9sKkzKO3hP6LNySO+us7i1RWc3P5oqjnzsiHNErhB6QUghIQs4nsMuSbEKbyhZkdMEPxSbDU6kU4t3Y/W4o1zqSOmEPvYRmLEet8TZwWGbmGIaM+pcWJ8fxlwiz8Xcwvq5Dj0AzT3iyqd3obwwH8MP04WTv72ntw8x91kmkrtSakq2EYxRiAw19B4W59A3FUAGhjnm5+8TOspZHbchrtF+aVcKq/FXBxEXR24QkaK4gH1pc7jvM5T+pp2QfC5hWdHV03uxda8V/SD3fyk=
 
 script:
 - docker build --target test --tag todo-app:test .
 - docker run --env SECRET_KEY --env SECRET_TOKEN --env BOARD_ID todo-app:test tests
```

Comment:
I realise this line is from module 7 but non-e2e tests shouldn't need the real secret key and token. They should use the .env.test values instead. It's always a good idea to minimise the access to real secrets.